hbase-server/src/main/resources/hbase-webapps/master/hbck.jsp

```diff
@@ -110,7 +110,7 @@
 
   <table class="table table-striped">
     <tr>
-      <th>Region</th>
+      <th>Region Encoded Name</th>
```

Comment:
I think through the path on fs we can know the table the region belongs to, but what about the start key? We can not get the start key through the encoded name?

Reply:
Need change more code if we want start key here. And maybe a issue in future. Now all report is based on region encoded name. And the HBCK2 tool use encoded name as parameter too.